src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -84,4 +94,56 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         }
     }
     */
+
+    bool Profiles::_IsCustomFontWeight(const Windows::UI::Text::FontWeight& weight)
+    {
+        if (weight == FontWeights::Thin() ||
+            weight == FontWeights::ExtraLight() ||
+            weight == FontWeights::Light() ||
+            weight == FontWeights::SemiLight() ||
+            weight == FontWeights::Normal() ||
+            weight == FontWeights::Medium() ||
+            weight == FontWeights::SemiBold() ||
+            weight == FontWeights::Bold() ||
+            weight == FontWeights::ExtraBold() ||
+            weight == FontWeights::Black() ||
+            weight == FontWeights::ExtraBlack())
+        {
+            return false;
+        }
+        else
+        {
+            return true;
+        }
+    }
+
+    IInspectable Profiles::CurrentFontWeight()
+    {
+        // if no value was found, we have a custom value, and we show the slider.
+        const auto maybeEnumEntry{ _FontWeightMap.TryLookup(_State.Profile().FontWeight().Weight) };
+        FontWeightSlider().Visibility(maybeEnumEntry ? Visibility::Collapsed : Visibility::Visible);
+        return winrt::box_value<Editor::EnumEntry>(maybeEnumEntry ? maybeEnumEntry : CustomFontWeight);
```

Comment:
you sure should not have to `box_value` this.

Reply:
(because you're just returning an EnumEntry, which is an IInspectable, directly.)